exetera/bin/exetera

```diff
@@ -37,22 +36,24 @@ def main():
 
     parser_import = subparsers.add_parser('import')
     parser_import.add_argument('-te', '--territories', default=None,
-                            help=('The territory/territories to filter the dataset on '
+                               help=('The territory/territories to filter the dataset on '
                                     '(runs on all territories if not set)'))
+    parser_import.add_argument('-ds', '--dataset', required=True,
+                               help='Open a dataset, creating it if it doesn’t exist')
     parser_import.add_argument('-s', '--schema', required=True,
-                            help='The location and name of the schema file')
+                               help='The location and name of the schema file')
     parser_import.add_argument('-i', '--inputs', required=True)
     parser_import.add_argument('-o', '--output_hdf5',
-                            help='The location and name of the output hdf5 file. If this is an existing file, it will'
+                               help='The location and name of the output hdf5 file. If this is an existing file, it will'
                                     ' be overwritten')
-    parser_import.add_argument('-w', '--overwrite', action='store_true',
-                            help="If set, overwrites an existing project rather than appending to it")
+    parser_import.add_argument('-w', '--overwrite', action='store_false',
```

Comment:
Why is this changing back to 'store_false'?

Reply:
I think overwrite should be defaulted to be True without adding -w in the command line. 
If we set the action to 'store_true', then we need specifically adding -w, no?

Reply:
The main problem case is that people run the importer script without updating names and accidentally delete two hours worth of importing. This prevents that from happening quite so easily. -w should force the overwrite and not -w should fail if there is an existing target

Reply:
Ah, thanks for explaining. Make sense.